src/v2/Components/Artwork/Details.tsx

```diff
@@ -131,17 +131,11 @@ export class Details extends React.Component<Props, null> {
     }
   }
 
-  saleLine() {
```

Comment:
This method wasn't being used.

Reply:
I wonder if this stuff was moved to MP 🤔 The details component is used all over in our artwork bricks so just wanna double-check / confirm this isn't being removed erroneously 👍

Reply:
Different parts of the sale message logic are duplicated all over our services (Gravity, Metaphysics, Eigen, Force...) and unfortunately there's no single source of truth. In this particular case though there's another method `saleInfoline` which is actually being used in this component.